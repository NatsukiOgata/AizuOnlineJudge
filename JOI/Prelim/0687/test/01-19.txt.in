100
999 628 658 861 991 843 808 669 474 790 963 408 929 422 932 755 934 736 884 500 886 798 426 994 956 942 489 517 860 641 967 938 978 620 844 987 825 750 885 696 1000 413 955 661 766 816 879 989 446 964 504 575 406 414 863 747 692 785 881 537 814 990 416 890 479 601 619 403 600 407 847 889 648 695 614 828 626 509 593 566 778 589 511 679 758 530 701 901 788 571 727 833 569 455 485 921 855 436 839 786
