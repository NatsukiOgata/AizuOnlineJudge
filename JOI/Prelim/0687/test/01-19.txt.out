31183
40039
